templates/category.html

```diff
@@ -13,27 +13,31 @@
            <div class="header-items"><a href="{{ url_for('users') }}">Users</a></div>
        </nav>
 
-    <div>
-       
-        <h1>{{ category_title }}</h1>
-    <div>
 
     <div>
-        {% for book in category %}
-        
-        <div class="single-category">
-            <div class="book-image"><a href="{{ url_for('book', book_id=book['id']) }}"><img src="{{ book['image'] }}" alt="{{ book['title'] }}"></a></div>
-
+       {% for category_name in category %}
```

Comment:
I'm a little confused. If you're returning a single category, you shouldn't have to iterate through them.

Reply:
I see the issue:

https://github.com/amymhaddad/upcase_booklist_app/pull/15/files#diff-3f41e546893dc64b71aaacad12cad815R128

This will return `.all()` implicitly, which is a list. To return a single value, add `.first()` to the end of the query. Then you won't need the for loop. Which will make more sense since this page only displays a single category.